runner/sql.go

```diff
@@ -329,15 +334,21 @@ func importPanel(
 	panel panelToImport,
 	qt quoteType,
 	panelResultLoader func(string, string) (chan map[string]any, error),
+	cacheEnabled bool,
 ) error {
 	var ddlColumns []string
 	for _, c := range panel.columns {
 		ddlColumns = append(ddlColumns, quote(c.name, qt.identifier)+" "+c.kind)
 	}
 
+	tableType := "TEMPORARY TABLE"
+	if cacheEnabled { // indicates that the sqlite file is not present.
```

Comment:
The comment doesn't make sense. I think it's clearer here without a comment unless I'm misunderstanding

Reply:
hmm yeah it does make more sense without it.